CHANGES.txt

```diff
@@ -38,6 +38,11 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       User-facing behavior does not change with this fix (GH Issue #3726).
     - Fix occasional test failures caused by not being able to find a file or directory fixture
       when running multiple tests with multiple jobs.
+    - Added support for a new command-line parameter `--hash-format` to override the default
```

Comment:
Add a note that this changes the sconsign file name.  Also add this blurb to RELEASE.txt

Reply:
Looks like this already had a mention saying `For all hash formats other than the default of `md5`, the SConsign database will include the name of the hash format.` But I also added an example here.

I also removed a line I had previously: `In addition, the default Decider is now called `content` instead of `MD5`.` I did this because I don't think it's useful and I can't remember why I added it. A developer reading that will think "uh oh, MD5 decider is deprecated" but I didn't actually deprecate it in this change. I didn't think this sentence was actionable so I removed it. If you think we should deprecate `env.Decider('MD5')` we can do it in a follow-up change.

I updated 2 sections of CHANGES.txt to describe (1) the new parameter and (2) the fact that `--md5-chunksize` is deprecated and people should use `--hash-chunksize` instead.